.github/renovate.json

```diff
@@ -367,6 +367,27 @@
       "automerge": true,
       "enabled": true,
       "ignoreUnstable": false
+    },
+    {
+      "matchPackageNames": [
+        "azuremonitor/containerinsights/ciprod/prometheus-collector/images"
```

Comment:
@Devinwong we didn't support this image yet ?

Reply:
we did. This is already in renovate.json

Reply:
It is but by default it's only for patches and we would like the automatic PRs for the minor version too